src/main/java/com/connexta/store/adaptors/S3StorageAdaptor.java

```diff
@@ -40,6 +42,33 @@ public S3StorageAdaptor(@NotNull final AmazonS3 amazonS3, @NotBlank final String
     this.bucket = bucket;
   }
 
+  @Override
+  public void store(
```

Comment:
Combine this store method with the one below. We don't have a need to store differently for metacards, files, and irms.

Reply:
What would the mediatype of a metacard be? `text/xml`?

Reply:
I see elsewhere we have application xml...

Reply:
> The registration information for text/xml-external-parsed-entity is
>    in all respects the same as that given for application/xml-external-
>    parsed-entity above (Section 9.3), except that the "Type name" is
>    "text".
https://tools.ietf.org/html/rfc7303

Reply:
I think it's always application/xml. You can debug sending a postman request and double check though. Or just guess application/xml and we can fix it later.